packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -1315,6 +1315,28 @@ void main() {
     expect(tester.takeException(), isNull);
   });
 
+  // Regression test for https://github.com/flutter/flutter/issues/154532.
+  testWidgets('Keyboard navigation does not throw when no entries match the filter',
+    (WidgetTester tester) async {
+      await tester.pumpWidget(MaterialApp(
+        home: Scaffold(
+          body: DropdownMenu<TestMenu>(
+            enableFilter: true,
+            dropdownMenuEntries: menuChildren,
+          ),
+        ),
+      ));
+      await tester.tap(find.byType(DropdownMenu<TestMenu>));
+      await tester.pump();
+      await tester.enterText(find.byType(TextField).first, 'No match');
+      await tester.pump();
+      await tester.sendKeyEvent(LogicalKeyboardKey.arrowDown);
+      await tester.pump();
+      await tester.enterText(find.byType(TextField).first, 'No match 2');
+      await tester.pump();
+      expect(tester.takeException(), isNull);
+  }, variant: TargetPlatformVariant.desktop());
```

Comment:
tiny whitespace nit:

```suggestion
    },
    variant: TargetPlatformVariant.desktop(),
  );
```

Reply:
Majority of the tests in the framework with `TargetPlatformVariant`  are using 

```}, variant: TargetPlatformVariant.desktop());```

 I want to keep it the same format

Reply:
Fair point!

<br>

Here are 2 good options:

```dart
  testWidgets('description', (WidgetTester tester) async {
    // ...
  }, variant: TargetPlatformVariant.desktop());
```

```dart
  testWidgets(
    'description',
    (WidgetTester tester) async {
      // ...
    },
    variant: TargetPlatformVariant.desktop(),
  );
```

I'd be happy with either one.